projects/js-packages/connection/state/actions.jsx

```diff
@@ -87,6 +103,32 @@ function* registerSite( { registrationNonce, redirectUri } ) {
 	}
 }
 
+/**
+ * Side effect action which will fetch a new list of connectedPlugins from the server
+ *
+ * @param {object}   store - Redux store.
+ * @param store.dispatch
+ * @returns {Promise}      - Promise which resolves when connectedPlugins is updated.
+ */
+function requestAndUpdateConnectedPlugins( { dispatch } ) {
+	return new Promise( resolve => {
```

Comment:
aren't we going to handle when something get fail?

Reply:
this isn't critical information. I guess we can ignore if the request fails... but we could explicitly ignore it

Reply:
👍